analytic_entries_report_fiscalyear/tests/test_analytic_entries_report_fiscalyear.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp.tests.common import TransactionCase
+
+
+class TestAnalyticEntriesReportFiscalyear(TransactionCase):
+    def test_analytic_entries_report_fiscalyear(self):
+        self.env['analytic.entries.report'].read_group(
```

Comment:
I have noticed this now fails in 2019 because of a missing fiscal year. Is there any standard way to enforce a fiscal year to exist before starting the test?

Reply:
the account module creates one for the current fiscal year. If we have problems with that, that's caused by our runbot which recycles old databases for performance reasons